metadata/2025-06-12-disable-concurrent-fa-balance/disable_concurrent_fungible_balance.json

```diff
@@ -0,0 +1,6 @@
+{
+  "title": "Disable CONCURRENT_FUNGIBLE_BALANCE",
+  "description": "To prevent issues before the fix, we should consider disabling the CONCURRENT_FUNGIBLE_BALANCE feature flag ahead of the coin-to-fungible store rollout.",
```

Comment:
@lightmark could we give a bit more detail on the issue?  this sounds very fuzzy